Global/convert-markdown.ini

```
@@ -0,0 +1,83 @@
+[Command]
+Name=Convert Markdown to ...
+Command="
+    copyq:
+    // # get input text to be converted
+    
+    var markdown = str(input());
+    if (!markdown) {
+        copy();
+        markdown = clipboard();
+    }
+    
+    // # get conversion options from user
+    
+    var renderers = {
+        'HTML': 'mistletoe.html_renderer.HTMLRenderer',
+        'HTML + code highlighting': 'contrib.pygments_renderer.PygmentsRenderer', // requires: `pip3 install pygments`
+        'HTML + GitHub Wiki': 'contrib.github_wiki.GithubWikiRenderer',
+        'HTML + MathJax': 'contrib.mathjax.MathJaxRenderer',
+        'Jira': 'contrib.jira_renderer.JIRARenderer',
+        'JSON (AST)': 'mistletoe.ast_renderer.ASTRenderer',
+        'LaTeX': 'mistletoe.latex_renderer.LaTeXRenderer',
+    }
+    
+    var settingsPrefix = 'mdConv.';
```

Using `convert-markdown/` would be a bit clearer and using the `/` separator creates group `convert-markdown` in the config file (`~/.config/copyq/copyq-scripts.ini`).

---

I didn't know that, thanks for the tip! I think I will notice this as part of hluk/copyq#1424. And possibly fix it right away in here:

```suggestion
    var settingsPrefix = 'convert-markdown/';
```